icons/README.md

```diff
@@ -0,0 +1,2 @@
+# Icons
+unplug by tezar tantular from <a href="https://thenounproject.com/browse/icons/term/unplug/" target="_blank" title="unplug Icons">Noun Project</a> (CC BY 3.0)
```

Comment:
Why html link here instead of markdown link?
does `[https://blabla](Noun Project)` not work here?

Reply:
Honestly, haven't even looked at it. Just mindlessly pressed the button and CTRL+V
![image](https://github.com/randovania/open-dread-rando/assets/117127188/da51db23-758c-42f8-8132-87c47e64fe58)

Reply:
Changed it.